exe/stepmod-annotate-all

```diff
@@ -2,8 +2,10 @@
 
 require "concurrent"
 require "stepmod/utils/stepmod_file_annotator"
+require "fileutils"
 
 stepmod_dir = ARGV.first || Dir.pwd
+stepmod_dir = Pathname.new(stepmod_dir.gsub("\\", "/"))
```

Comment:
@ronaldtse From reading the documentation it seems like `File.expand_path` only expands the path but if user provides an input like `stepmod-annotate-all D:\Work\Metanorma\repositories\iso-10303-stepmod_test_utils\` it will not convert the `\` to `/` and I think for that we have to manually change it using `gsub`.